styles/twitch/catppuccin.user.css

```
@@ -711,6 +711,382 @@
+    }
+
+    tbody tr td {
+      border-color: @surface0;
+    }
+
+    tbody tr:nth-child(odd) {
+      background: @base !important;
+    }
+    tbody tr:nth-child(even) {
+      background: @mantle !important;
+    }
+
+    h1,
+    h2,
+    h3,
+    h4,
+    h5,
+    h6,
+    hr {
+      color: @text !important;
+      border-color: @surface0 !important;
+    }
+
+    .right-code {
+      p {
+        color: @subtext0;
+      }
+    }
+
+    blockquote {
+      border-color: @accent-color;
+      background: @mantle;
+    }
+
+    .content,
+    .main,
+    .topics-list,
+    .doc-content {
+      background: @base;
+      border-color: @base !important;
+    }
+
+    code {
+      border-color: @accent-color !important;
+      background: fade(@accent-color, 5%) !important;
+      color: @text !important;
+    }
+
+    /* Code */
+    .right-code pre,
+    pre.highlight,
+    pre {
+      border-color: @mantle !important;
+      background: @mantle !important;
+      code {
+        background: transparent !important;
+        color: @text !important;
+        /* Syntax highlighting */
+        .c,
+        .c1 {
+          color: @surface1 !important;
+        }
+        .p,
+        .w,
+        .nv {
+          color: @text !important;
+        }
+        .nt,
+        .o {
+          color: @red !important;
+        }
+        .s1,
+        .s2 {
+          color: @yellow !important;
+        }
+        .se,
+        .mi {
+          color: @mauve !important;
+        }
+        .kc,
+        .kt,
+        .nb {
+          color: @sapphire !important;
+        }
+        .k,
+        .na,
+        .kn,
+        .kr,
+        .kd {
+          color: @blue !important;
+        }
+        .nn,
+        .m,
+        .bp {
+          color: @teal !important;
+        }
+        .nc,
+        .nf,
+        .cp {
+          color: @green !important;
+        }
+        .s {
+          color: @peach !important;
+        }
+      }
+    }
+
+    /* Pills */
+    .pill-new {
+      color: @base !important;
+      background: @accent-color !important;
+    }
+    .pill-beta {
+      color: @base !important;
+      background: @yellow !important;
+    }
+
+    a {
+      color: @accent-color;
+    }
+
+    /* Navbar */
+    .sidebar {
+      background: @mantle !important;
+      dl {
+        border-color: @base !important;
+      }
+      dt a,
+      dl dd a {
+        color: @text !important;
+      }
+      dl dd a.active-highlight {
+        color: @accent-color !important;
+      }
+      dl dd a.active-highlight::before {
+        border-left-color: @accent-color !important;
+      }
+      dt a:hover {
+        background: @base !important;
+      }
+
+      /* Search */
+
+      input {
+        color: @text;
+      }
+      input::placeholder {
+        color: @subtext0;
+      }
+      input:focus {
+        background: @crust !important;
+        box-shadow: 0 0 3px 1px fade(@accent-color, 70%);
+        color: @text;
+      }
```

```suggestion
      input {
        color: @text;
        &::placeholder {
          color: @subtext0;
        }
        &:focus {
          background: @crust !important;
          box-shadow: 0 0 3px 1px fade(@accent-color, 70%);
          color: @text;
        }
      }
```
Does `input::placeholder` need to be set here? Our base styles should cover it.